manifests/params.pp

```diff
@@ -121,6 +128,17 @@
   validate_bool($rhn)
   validate_hash($descriptions)
 
+  # validate email addresses based on regex found in `is_email_address`
+  # in newer stdlib versions.  If metadata.json updated to require
+  # *4.12.0* or newer, the `validate_re` calls can be replaced by
+  # `validate_email_address`.
```

Comment:
@jearls Thanks for your PR!  I'd be OK with updating the dependency on stdlib to 4.12.0